rules/block-us.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "id": 1,
+    "priority": 1,
+    "action": { 
+      "type": "modifyHeaders",
+      "requestHeaders": [{
+        "header": "user-agent",
+        "operation": "remove"
+      }]
+    },
+    "condition": {"excludedResourceTypes": ["main_frame"]}
```

Comment:
Seems like both `excludedResourceTypes` and `resourceTypes` needs to be specified in order to block all type of the requests, otherwise [when `excludedResourceTypes` is skipped](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/#property-RuleCondition-excludedResourceTypes), then `main_frame` is skipped.